datasets/doc/source/how-to-use-with-local-data.rst

```diff
@@ -44,6 +44,8 @@ CSV
     "can-have-more-splits": ...
   }
   dataset = load_dataset("csv", data_files=data_files)
+  # if data_files is a dictionary choose the split e.g. "train"
+  # dataset = load_dataset("csv", data_files=data_files, split="train")
```

Comment:
I'm unsure why this is needed here. Wouldn't it be more relevant to mention when constructing the partitioner? 

The comment seems to indicate that when what gets passes is a dic, you need to specify a split. But this is optional.